docs/db/changelog/changesets/powerauth-java-server/1.9.x/20240910-commit-phase.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.9.x/20240910-commit-phase.xml" author="Roman Strobl">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="pa_activation" columnName="commit_phase"/>
+            </not>
+        </preConditions>
+        <comment>Add commit_phase column to pa_activation table.</comment>
+        <addColumn tableName="pa_activation">
```

Comment:
Joy of MSSQL. If we need to have this for old records (can be the case) we need to use either update after the column is added, or specific add of the column for MSSQL:

`ALTER TABLE pa_activation ADD protocol commit_phase integer CONSTRAINT DF_pa_activation_commit_phase DEFAULT 0 WITH VALUES;`

If we do not need to care about already created records we can omit this.

Reply:
We can keep the `null` values for existing rows, this case is supported.